ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1568,3 +1568,15 @@ IE_GPU.evaluate_1D_gather_elements_negative_test
 IE_GPU.evaluate_2D_gather_elements_negative_test
 IE_GPU.evaluate_2D_gather_elements_2x2x1_data_float32
 IE_GPU.evaluate_4D_gather_elements_3x2x2x2_indices_int64
+
+# not implemented in CPU and GPU
```

Comment:
Since both plugins lack support and the unit tests are not executed on GPU in CI, you can remove the `IE_CPU.` and `IE_GPU.` prefixes and disable the tests by adding jsut their names once in this file.